docs/ops/homeserver-front-domain-cutover.md

```
@@ -31,6 +31,38 @@
+   > `ENV`는 builder 스테이지를 넘지 않아 런타임 스테이지에는 존재하지 않고, 좋은 digest와 나쁜
+   > digest 모두 빈 출력 + `exit 1`이다(실측). **산출물을 봐야 한다.**
+   > ```bash
+   > docker run --rm --entrypoint sh ghcr.io/aquilaxk/aquila-blog-front@sha256:<digest> -c '
+   >   set -e
+   >   grep -rq "https://blog\.aquilaxk\.site"     /app/.next
+   >   grep -rq "https://api\.blog\.aquilaxk\.site" /app/.next
+   >   ! grep -rq "https://www\.aquilaxk\.site"    /app/.next
+   > '; echo "gate exit=$?"   # 0이어야 쓸 수 있다
+   > ```
+   > 실측(좋은 이미지 = 이 PR 기준 로컬 빌드 / 나쁜 이미지 = `sha-86e5547a5`):
+   > ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)